filesystem.go

```diff
@@ -152,11 +152,6 @@ func (fs *filesystem) createAllResources(pod Pod) error {
 }
 
 func (fs *filesystem) storeFile(path string, data interface{}) error {
-	_, err := os.Stat(path)
```

Comment:
What do you mean, you try to store everything to a temporary file and if everything succeed, in the end of the function you move the content of this temp file to the one we are targeting ?